lib/config.js

```diff
@@ -53,7 +53,8 @@ config.cronIntervalExternalGraderLoadSec = 8;
 config.cronIntervalServerLoadSec = 8;
 config.cronIntervalServerUsageSec = 8;
 config.cronIntervalCalculateAssessmentQuestionStatsSec = 10 * 60;
-config.cronIntervalStopStaleWorkspacesSec = 10 * 60;
+config.cronIntervalStopStaleWorkspacesSec = 60;
+config.cronIntervalWarnStaleWorkspacesSec = 60;
```

Comment:
Are these crons guaranteed to be in sync?

Reply:
Not guaranteed. Multiple servers can run cron jobs and they can diverge over time. Is it important that these stay in sync?

Reply:
Both `stop` and `warn` do their calculations based on `launched_at`, but we don't actually stop those workspaces at exactly `launched_at + config.launchedTimeout`. I think the actual stopping occurs whenever the next `stop` cron happens to fire, so basically `warn`'s estimate is always off -- first because `stop` only occurs at given intervals and further if `stop`/`warn` get desync'ed.

I lowered the sproc intervals to 60 sec, so I think it will be off by at most 59 sec. Maybe that's good enough given there should be no data loss here, just interruption of workflow?

Reply:
Yep, I think this is fine. I like the way you said "_less than_ XX min" in the warning message.